### PR TITLE
Only limit drag resizes

### PR DESCRIPTION
**Pull request by phonetic112**
Fixes https://github.com/hyprwm/Hyprland/issues/5157

### CODE REVIEW

**Review by JunaidQrysh — APPROVED** [review]

**Review by JunaidQrysh — APPROVED** [review]
Can confirm it fixes #5157

**Review by vaxerski — DISMISSED** [review]

File: src/layout/IHyprLayout.cpp [hunk 1]
```diff
@@ -355,7 +355,7 @@ void IHyprLayout::onMouseMove(const Vector2D& mousePos) {
         canSkipUpdate = std::clamp(MSMONITOR - TIMERDELTA, 0.0, MSMONITOR) > totalMs * 1.0 / m_iMouseMoveEventCount;
     }
 
-    if ((abs(TICKDELTA.x) < 1.f && abs(TICKDELTA.y) < 1.f) || (TIMERDELTA < MSMONITOR && canSkipUpdate))
+    if (((abs(TICKDELTA.x) < 1.f && abs(TICKDELTA.y) < 1.f) || (TIMERDELTA < MSMONITOR && canSkipUpdate)) && g_pInputManager->dragMode != MBIND_MOVE)
```
Comment (vaxerski) [inline src/layout/IHyprLayout.cpp:358]: actually, no need to update if tickdelta is less than 1,1, so can you leave that in?

Comment (phonetic112) [reply src/layout/IHyprLayout.cpp:358]: wdym?

Comment (vaxerski) [reply src/layout/IHyprLayout.cpp:358]: if we are moving, rn, even if delta.x < 1 and delta.y < 1 we still process it, which is redundant

Comment (vaxerski) [reply src/layout/IHyprLayout.cpp:358]: tldr:
```cpp
if (((abs(TICKDELTA.x) < 1.f && abs(TICKDELTA.y) < 1.f) || (TIMERDELTA < MSMONITOR && canSkipUpdate)) && g_pInputManager->dragMode != MBIND_MOVE)
```
into 
```cpp
if ((abs(TICKDELTA.x) < 1.f && abs(TICKDELTA.y) < 1.f) || (TIMERDELTA < MSMONITOR && canSkipUpdate && g_pInputManager->dragMode != MBIND_MOVE))
```

Comment (phonetic112) [reply src/layout/IHyprLayout.cpp]: Done

**Review by vaxerski — APPROVED** [review]
cute

**Pull request merged**